FusionIIIT/applications/placement_cell/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.1.5 on 2023-04-06 12:31
+# Generated by Django 3.1.5 on 2023-03-15 18:53
 
```

Comment:
You don't need to push this file , while you push you add files in your PR using git add ignore this file.